packages/react-router-dom/__tests__/data-browser-router-test.tsx

```diff
@@ -522,62 +507,50 @@ function testDomRouter(
         return <h1>{data?.message}</h1>;
       }
 
-      expect(getHtml(container)).toMatchInlineSnapshot(`
-        "<div>
-          <div>
-            <a
-              href=\\"/bar\\"
```

Comment:
Removed the href-specific parts of these snapshots since href-generation is tested elsewhere.  This way we can share these functional tests between `createBrowserRouter` and `createHashRouter`